less/components/index.less

```diff
@@ -8,3 +8,4 @@
 @import "page";
 @import "page-end-cta";
 @import "home-page";
+@import "clubs-page";
```

Comment:
Huh, what happened to this? Did it go away in an earlier commit but we forgot to remove the `@import` until now?